notation/verifier.go

```diff
@@ -32,50 +32,48 @@ import (
 const (
 	notationVerifierType          = "notation"
 	notationSignatureArtifactType = "application/vnd.cncf.notary.signature"
-	maxManifestSizeLimit          = 4 * 1024 * 1024  // 4 MiB
-	maxBlobSizeLimit              = 32 * 1024 * 1024 // 32 MiB
 )
 
 // NewVerifierOptions contains the options for creating a new Notation verifier.
 type NewVerifierOptions struct {
-	// Name is the name of the verifier. Required.
+	// Name is the instance name of the verifier to be created. Required.
 	Name string
 
 	// TrustPolicyDoc is a trustpolicy.json document. It should follow the spec:
-	// https://github.com/notaryproject/notation-go/blob/release-1.3/verifier/trustpolicy/oci.go#L29
+	// https://github.com/notaryproject/notation-go/blob/v1.3.0/verifier/trustpolicy/oci.go#L29
 	// Required.
 	TrustPolicyDoc *trustpolicy.Document
 
 	// TrustStore manages the certificates in the trust store. It should
 	// implement the truststore.X509TrustStore interface:
-	// https://github.com/notaryproject/notation-go/blob/release-1.3/verifier/truststore/truststore.go#L52
+	// https://github.com/notaryproject/notation-go/blob/v1.3.0/verifier/truststore/truststore.go#L52
 	// Required.
 	TrustStore truststore.X509TrustStore
 
 	// PluginManager manages the plugins installed for Notation verifier. It
 	// should implement the plugin.Manager interface:
-	// https://github.com/notaryproject/notation-go/blob/release-1.3/plugin/manager.go#L33
+	// https://github.com/notaryproject/notation-go/blob/v1.3.0/plugin/manager.go#L33
 	// Optional.
 	PluginManager plugin.Manager
 }
 
 // Verifier is a ratify.Verifier implementation that verifies Notation
 // signatures.
 type Verifier struct {
-	name        string
-	sigVerifier notation.Verifier
+	name     string
+	verifier notation.Verifier
 }
 
 // NewVerifier creates a new Notation verifier.
-func NewVerifier(opts *NewVerifierOptions) (ratify.Verifier, error) {
-	verifier, err := notationVerifier.New(opts.TrustPolicyDoc, opts.TrustStore, opts.PluginManager)
+func NewVerifier(opts *NewVerifierOptions) (*Verifier, error) {
+	verifier, err := verifier.New(opts.TrustPolicyDoc, opts.TrustStore, opts.PluginManager)
```

Comment:
nit: try not use a variable name same as the package name.
```suggestion
	v, err := verifier.New(opts.TrustPolicyDoc, opts.TrustStore, opts.PluginManager)
```